zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/archiver/OpenSearchRepositoryIT.java

```diff
@@ -21,19 +22,24 @@
 import java.time.Duration;
 import java.util.List;
 import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
 import org.apache.http.HttpHost;
 import org.junit.jupiter.api.Test;
 import org.opensearch.client.RestClient;
 import org.opensearch.client.json.jackson.JacksonJsonpMapper;
 import org.opensearch.client.opensearch.OpenSearchAsyncClient;
 import org.opensearch.client.opensearch.OpenSearchClient;
 import org.opensearch.client.opensearch.core.search.Hit;
+import org.opensearch.client.opensearch.generic.OpenSearchGenericClient;
+import org.opensearch.client.opensearch.generic.Requests;
 import org.opensearch.client.transport.rest_client.RestClientTransport;
 import org.opensearch.testcontainers.OpensearchContainer;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.testcontainers.junit.jupiter.Container;
 import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.shaded.org.awaitility.Awaitility;
```

Comment:
:x: Be careful not to import shaded dependencies, but the actual dependency. It's a very easy mistake to make with an IDE :sweat_smile: 

> [!Note]
> In case you never came across that, shaded dependencies are dependencies which have been fully merged into another one - think putting a JAR into another JAR - and then all their classes are relocated under a nested package. So instead of `org.awaitility.Awaitility`, you get `org.testcontainers.shaded.org.awaitility.Awaitility`.

It's better to use the real dependency, so we can properly keep it up to date and detect CVEs.

Reply:
Good catch 🙌